targettests/TargetConfig/RegressionValidation/anyon.config

```diff
@@ -0,0 +1,52 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+#  RUN: cudaq-target-conf -o %t %cudaq_target_dir/anyon.yml && cat %t | FileCheck %s
+
+# Tell DefaultQuantumPlatform what QPU subtype to use
+# CHECK-DAG: PLATFORM_QPU=remote_rest
+
+# Tell NVQ++ to generate glue code to set the target backend name
+# CHECK-DAG: GEN_TARGET_BACKEND=true
+
+# Add the rest-qpu library to the link list
+# CHECK-DAG: LINKLIBS="${LINKLIBS} -lcudaq-rest-qpu"
+
+# Define the lowering pipeline. telegraph-8q has an 8-qubit ring topology, so mapping
+# uses ring(8).
+# Toshiko uses a Kagome lattice with 2-3 connectivity per qubit
```

Comment:
I'm guessing this comment isn't relevant for this file? (Toshiko is for another hw provider.) If so, would you please remove it?

Reply:
updated with information on Berkeley-25q to replace the line in question